orchagent/vnetorch.cpp

```diff
@@ -97,6 +99,7 @@ bool VNetVrfObject::createObj(vector<sai_attribute_t>& attrs)
                            vnet_name_.c_str(), status);
             throw std::runtime_error("Failed to create VR object");
         }
+        gFlowCounterRouteOrch->onAddVR(router_id);
```

Comment:
Vnet is a special case. I don't think we should add vnet routes to flowcounters. May be it can be considered for later phase

Reply:
I prefer to keep this as this is required in the HLD review. Thanks.